docs/credits.md

```diff
@@ -1031,6 +1033,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | thunky                                            | Mathias Buus Madsen                                                       | [https://github.com/mafintosh/thunky](https://github.com/mafintosh/thunky)                                                                                                                                                  | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | timers-browserify                                 | J. Ryan Stinnett                                                          | [https://github.com/jryans/timers-browserify](https://github.com/jryans/timers-browserify)                                                                                                                                  | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | TimeZoneConverter                                 | Matt Johnson                                                              | [https://github.com/mj1856/TimeZoneConverter](https://github.com/mj1856/TimeZoneConverter)                                                                                                                                  | [MIT](https://github.com/mj1856/TimeZoneConverter/blob/master/LICENSE.txt)                                          |
+| TimeZoneNames                                     | Matt Johnson                                                              | [https://github.com/mj1856/TimeZoneNames](https://github.com/mj1856/TimeZoneNames)                                                                                                                                          | [Undefined]()                                                                                                       |
```

Comment:
Interesting @matt-richardson this is using the MIT License. https://github.com/mj1856/TimeZoneNames/blob/master/LICENSE

Perhaps because it's missing the `.txt` extension?